### ISSUE TITLE
nerdtree sidebar auto-closing on file open with recent update: can't disable via NERDTreeQuitOnOpen=0

### ISSUE THREAD

**Issue by IAXES** [preservim/nerdtree#1223]
#### Self-Diagnosis

- [X] I have searched the [issues](https://github.com/scrooloose/nerdtree/issues) for an answer to my question.
- [X] I have reviewed the NERDTree documentation. `:h NERDTree`
- [X] I have reviewed the [Wiki](https://github.com/scrooloose/nerdtree/wiki).
- [x] I have searched the web for an answer to my question.

#### Environment (for bug reports)
- [X] Operating System: **Ubuntu 18.04 LTS 86_64**
- [X] Vim/Neovim version `:echo v:version`: **(802) - VIM - Vi IMproved 8.2, Included patches: 1-2486**
- [ ] NERDTree version, found on 1st line in NERDTree quickhelp `?`: **6.10.5**
- [ ] vimrc settings
    - [ ] NERDTree variables
    ```vim
    autocmd VimEnter * NERDTree
    " Just for testing to see if this was a case of the feature being turned on by default now; no effect
    let g:NERDTreeQuitOnOpen = 0 
    ```
    - Other NERDTree-dependent Plugins
        - [ ] jistr/vim-nerdtree-tabs
        - [ ] ryanoasis/vim-devicons
        - [ ] tiagofumo/vim-nerdtree-syntax-highlight
        - [ ] Xuyuanp/nerdtree-git-plugin
        - [ ] Others (specify): 
    - [] I've verified the issue occurs with only NERDTree installed.

#### Steps to Reproduce the Issue
1. Start `vim`.
2. Open a file w/ NERDtree.
3. NERDtree sidebar auto closes. Need to re-open with `NERDTreeToggle` each time (annoying).
4. Tried playing with the `NERDTreeQuitOnOpen` variable, as per other issue reports: no luck/effect.

#### Expected Result

NERDtree sidebar to remain open after using it to open a file/buffer. This only just started happening when I updated my `vundle` plugins a few days ago.

**Comment by lukexor** [comment]
Seems it was last working on 6.10.4.

Work around for now for vim-plug:

`Plug 'preservim/nerdtree', { 'on': 'NERDTreeToggle', 'tag': '6.10.4' }`

**Comment by IAXES** [comment]
Thanks. I reverted back to `6.10.4` in `vundle` via checking out that tag. Seems vundle doesn't support anchors/concrete-refs. Might have to switch to a different plugin manager soon. What do you use?

**Comment by PhilRunninger** [comment]
Duplicate of #1220. The fix is in 6.10.6. Update and try again.

**Comment by IAXES** [comment]
Confirmed working with latest HEAD rev: `3a9d533f3de86a43b69f6c47d3394c0d866fdb08`.

Thanks!
